src/main/java/ca/ciccc/SeatingChart.java

```diff
@@ -29,6 +29,16 @@
      */
     public SeatingChart(List<Student> studentList, int rows, int cols) {
         // TODO: Your code goes here.
+        seats = new Student[rows][cols];
+        int n = 0;
+        for (int i = 0; i < cols; i++) {
+            for (int j = 0; j < rows; j++) {
+                if (n < studentList.size()) {
+                    seats[j][i] = studentList.get(n);
+                    n++;
+                }
```

Comment:
if (n >= studentList.size()) {
    // should break all loop
    break;
}
seats[j][i] = studentList.get(n);
n++;